cmd/main.go

```diff
@@ -146,6 +146,7 @@ var Options struct {
 	HTTPListenPort                 string        `envconfig:"HTTP_LISTEN_PORT" default:""`
 	AllowConvergedFlow             bool          `envconfig:"ALLOW_COVERGED_FLOW" default:"true"`
 	IronicIgnitionBuilderConfig    ignition.IronicIgniotionBuilderConfig
+	ClusterTLSCertsDir             string `envconfig:"ASSISTED_SERVICE_CLUSTER_TLS_CERTS_DIR" default:""`
```

Comment:
Some documentation / comment about what the purpose of this configuration environment variable could be nice. I don't feel it's obvious at all from the name

Reply:
Also a warning about the context in which it needs to be used - it would be pretty horrible, IIUC, to use this in production / ACM

Reply:
Also the name should be modified to reflect this, maybe with the use of words like "FORCE" or "OVERRIDE"